docs/UserGuide.md

```diff
@@ -360,169 +373,169 @@ Example: <br>
 Gives a **person / staff / supplier / maintainer** contact from your PoochPlanner a performance rating so that cafe owners can track the performance of their contacts.
 
 Format: <br>
-`/rate ; name : [name] ; rating : [rating value from 1-5]`
+`/rate ; name : [name] ; rating : [rating value from 0-5]`
 
 Example:
 * `/rate ; name : Poochie ; rating : 3`
 
-  The above command rates the contact named `Poochie` with a rating of `3`, provided `Poochie` exists in the contact list.
+  The above command rates the contact named `Poochie` with a rating of `3`, provided a contact with name `Poochie` exists in your contacts list.
 
 <br> 
 
 <div style="text-align:center;">
     <br>
     <img src="images/ug-images/command-images/ratecommand.png" alt="Add before" style="width:100%;"/>
-    <medium>Above is an example of rating a person contact named Janna :  </medium>
+    <medium>Above is an example of rating a person contact named "Janna":</medium>
     <br>
-    <medium>Input: `/rate ; name : Janna ; rating : 5` </medium>
+    <medium>Input: <code>/rate ; name : Janna ; rating : 5</code></medium>
     <br>
-    <medium>Output: Rate the contact named `Janna` with a rating of `5` in the contact list. </medium>
+    <medium>Output: Rates the contact named "Janna" with a rating of <code>5</code> in your contacts list.</medium>
     <br>
 </div>
 
 <br> 
 
-<div markdown="span" class="alert alert-warning">:exclamation: **Constraints:**<br>
+<div markdown="block" class="alert alert-warning">:exclamation: **Constraints:**<br>
 
-* Rating can only accept whole number values from 1 to 5 inclusive.<br>
+* `rating` can only accept whole number values from `0` to `5` inclusive.<br>
```

Comment:
is there a difference between ` and <code> ? why some commands use ` and some use <code>

Reply:
the html code tag is used within enclosing html tags because there is no markdown parser within html blocks that can parse traditional backticks